### ISSUE TITLE
Support for ANSI "color" escape sequences

### ISSUE THREAD

**Issue by rsynnest** [akavel/up#22]
Great tool! I have a non-critical feature request: It would be nice if `up` supported color codes such as those output by `grep --color=always`. 
Thanks!

**Comment by akavel** [comment]
Ah, you mean if `grep --color=always` is entered as a command in *up*? It never occurred to me, but that's a good idea indeed! Though I'm not quite sure how it could be handled in tcell...

**Comment by lordlycastle** [comment]
Yes this would be very cool. E.g. if you do `cat file | up` then `| ack bob` it would highlight all matches. Or keep colors of `jq` or color coded syntax highlighting when searching. 

Anyways it's still an amazing tool!!

**Comment by nkh** [comment]
@lordlycastle https://github.com/paoloantinori/hhighlighter highlights multiple strings, invaluable sometimes.

**Comment by nkh** [comment]
filtering files looks sad without colors

![screenshot_2023-06-16_11-17-19](https://github.com/akavel/up/assets/45409/dd8833c9-a31f-4f37-b532-4389e4a27a7d)
